atom/browser/api/atom_api_app.cc

```diff
@@ -47,7 +47,6 @@
 #include "content/public/browser/browser_thread.h"
 #include "content/public/browser/client_certificate_delegate.h"
 #include "content/public/browser/gpu_data_manager.h"
-#include "content/public/browser/memory_pressure_controller.h"
```

Comment:
memory_pressure_controller.h was deleted in https://chromium.googlesource.com/chromium/src/+/37c4e81eb4dfa713a06164dbce721710b92b2ca9
I'll explicitly include "base/memory/memory_pressure_listener.h", I thought this was checked by cpplint